src/sqliteformat.cpp

```diff
@@ -2206,6 +2366,13 @@ bool SqliteFormat::Private::selectCalendarProperties(Notebook::Ptr notebook)
     return success;
 }
 
+QDateTime SqliteFormat::deleted(const KCalendarCore::Incidence &incidence)
```

Comment:
Perhaps name like deletedDate(), deletionDate() or something like that could be a bit better here and on SingleSqlitebackend? Would match the existing SqliteStorage::incidenceDeletedDate() too.

Reply:
You're right. I put `deleted()` because KCalendarCore::Incidence is having `created()`. But it's too generic as a name and since it's a static method and not a member, `deletedDate()` sounds better.